src/app/(home)/[not-found]/page.module.css

```diff
@@ -0,0 +1,111 @@
+.notFoundPageWrap {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	width: 100vw;
+	height: 100vh;
+	background-color: var(--color-silver-gray);
+}
+.notFoundPageContainer {
+	display: flex;
+}
+.notFoundPageContainer__image {
+	width: 409px;
+	height: 407px;
+}
+.titleContainer {
+	font-family: "Rubik";
+	font-weight: 400;
+	color: var(--color-dark);
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	width: 404px;
+	height: 382px;
+	margin-top: 10px;
+	margin-left: 17px;
```

Comment:
better to change the margin-left in this class to use the gap property in the notFoundPageContainer class